prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -133,3 +133,37 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
   # gcloud beta monitoring channels list --project=oss-prow
   notification_channels = ["projects/${var.project}/notificationChannels/${var.notification_channel_id}"]
 }
+
+resource "google_monitoring_alert_policy" "probers" {
+  project = var.project
+
+  display_name = "HostDown"
+  combiner     = "OR"
+  conditions {
+    display_name = "Host is unreachable"
+    condition_monitoring_query_language {
+      duration = "120s"
+      query    = <<-EOT
+      fetch uptime_url
+      | metric 'monitoring.googleapis.com/uptime_check/check_passed'
+      | filter (metric.check_id == 'blackbox-exp')
+      | align next_older(1m)
```

Comment:
The choice between `next_older` and `next_younger` doesn't matter here, right? You're just trying to grab a discrete value without actual aggregation?

Reply:
> The choice between `next_older` and `next_younger` doesn't matter here, right? You're just trying to grab a discrete value without actual aggregation?

this was adapted from auto-generated alerts from UI, where we can set an alert from `uptime check` page